### PR TITLE
students can view questions in view answerpaper

### PR DESCRIPTION
**Pull request by maheshgudi**
Students get to see the actual questions with their answers in their answer paper.
Modified/beautified the look on view_answerpaper page.
screenshot from 2016-09-07 15 36 20
screenshot from 2016-09-07 15 36 30

### CODE REVIEW

**Comment by ankitjavalkar** [comment]
Looks good.

File: yaksh/templates/yaksh/view_answerpaper.html [hunk 1]
```diff
@@ -24,29 +26,57 @@
...
+            <strong> Details: {{forloop.counter}}. {{ question.summary }} 
+            <span class="marks"> Mark(s): {{ question.points }} </span>
+            </strong>
+        </div>
+        <div class="panel-body">
+            <h5><u>Question:</u></h5> <strong>{{ question.description|safe }}</strong>
+            {% if question.type == "mcq" or question.type == "mcc" %}
+            <h5> <u>Choices:</u></h5>
+            {% for testcase in question.get_test_cases %}
+            <br/><strong>{{ forloop.counter }}. &nbsp{{ testcase.options }}</strong>
```
Comment (prathamesh920) [inline yaksh/templates/yaksh/view_answerpaper.html]: Its okay, but &nbsp not needed. No harm if it s there.

Comment (prathamesh920) [inline yaksh/templates/yaksh/view_answerpaper.html]: We can merge this.

Comment (maheshgudi) [inline yaksh/templates/yaksh/view_answerpaper.html]: resolved issues nonetheless

**Comment by prabhuramachandran** [comment]
It would be nice if the multiple choice was rendered better but we can change that later.  The traceback also shows in one line, it would be nice to render it nicely.

**Pull request merged**